package.json

```diff
@@ -6,58 +6,56 @@
 		"checkFormat": "npm run prettier -- --list-different",
 		"coveralls": "node node_modules/coveralls/bin/coveralls.js < coverage/lcov.info",
 		"format": "npm run prettier -- --write",
-		"jest": "cross-env NODE_ENV=test jest",
 		"lerna": "lerna bootstrap -- --no-optional --no-package-lock",
 		"link": "lerna run link",
-		"lint": "eslint packages/**/*.js examples/**/*.js scripts/**/*.js",
-		"pa11y": "pa11y-ci ./packages/clay-*/demos/a11y.html",
-		"prettier": "prettier-eslint packages/**/**/*.js examples/**/**/*.js scripts/**/**/*.js",
-		"start": "http-server . -p 4000",
-		"test": "npm run build && npm run jest && npm run pa11y",
+		"lint": "eslint \"**/*.{js,ts,tsx}\"",
+		"prettier": "prettier-eslint \"**/*.{js,ts,tsx}\"",
```

Comment:
You can safely use single quotes here if you want to avoid the backslash escaping.